vue-components/stories/ExtendedNumberInput.stories.ts

```diff
@@ -0,0 +1,83 @@
+import ExtendedNumberInput from '@/components/ExtendedNumberInput';
+import { Component } from 'vue';
+import {errorProp} from "../src/compositions/validatable";
+import {MenuItem} from "../src/components/MenuItem";
+import {ErrorProp} from "../dist/compositions/validatable";
+
+export default {
+    component: ExtendedNumberInput,
+    title: 'Wikibase Components/ExtendedNumberInput',
+};
+
+export function basic( args: { label: string, placeholder: string } ): Component {
+    return {
+        components: { ExtendedNumberInput },
+        data(): unknown {
+            return {
+                placeholder: args.placeholder,
+                label: args.label,
```

Comment:
You can drop those two data properties as they are already defined as props. That also takes care of the error messages in the console

Reply:
Oh I saw it, wanted to investigate but forgot. Thanks. Removed now.